### PR TITLE
update pinning comment about what defaults uses for protobuf

### PR DESCRIPTION
**Pull request by dougalsutherland**
Forgot to change this when I updated #411; sorry about that. Just to keep the comments marginally correct. :)

### CODE REVIEW

File: scripts/pin_the_slow_way.py [hunk 1]
```diff
@@ -80,7 +80,7 @@
           'pango': 'pango 1.40.*',  # 1.40.3
           'pixman': 'pixman 0.34.*',  # 0.34.0
           'proj4': 'proj4 4.9.3',  # 4.9.2
-          'protobuf': 'protobuf 3.4.*',  # 3.2.0
+          'protobuf': 'protobuf 3.4.*',  # 3.4.0
```
Comment (jakirkham) [inline scripts/pin_the_slow_way.py]: So this is what `defaults` is using now?

**Comment by dougalsutherland** [comment]
It's at least the newest version available in `defaults`. I didn't check which version any packages actually link to....

**Comment by jakirkham** [comment]
@mingwandroid, does this match up with `defaults` or is a different `protobuf` version in use currently?

**Comment by mingwandroid** [comment]
@jjhelmus is your go-to guy for all your `protobuf` enquiries. I think he's been working on it recently.

**Comment by jakirkham** [comment]
Friendly nudge @jjhelmus. 😉

**Comment by jjhelmus** [comment]
`defaults` is using protobuf 3.4.x on linux-64 and 3.2.x on the other platforms at the moment.  I've been working on getting 3.4.1 package out for all platforms but a failing unit test with the GCC 7.2.0 toolchain is holding this up.  

Having conda-forge pin protobuf 3.4.* is a big :+1: from me, `defaults` will be there soon.

**Comment by jakirkham** [comment]
Good to hear. Though this PR was just trying to fix the comment on what `defaults` is pinned to. My bad if we missed you on the previous PR. Should we be saying `3.2.*` or `3.3.*` for `defaults` then?

**Comment by jjhelmus** [comment]
In `defaults` protobuf is only used by the `caffe`, and `tensorflow` packages which do not use a uniform pinning.  I'm working on getting packages ready for both of these which use `protobuf` 3.4.x.

**Pull request merged**